### DISCUSSION TITLE
Per-User Hide Content

### DISCUSSION THREAD

**Discussion by Dan-613** [jellyfin/jellyfin#13011] [Q&A]
I'm thinking of working on this feature request.

https://features.jellyfin.org/posts/1072/let-the-user-hide-a-movie-or-tv-show

Let users hide movies and shows.

As this is the first time I'm considering look at handling a feature request, I was wondering if I could have some guidance as to the best place to store per-user hidden flags in the database.

Maybe a link table?

I'd love some guidance.
Also, if there is an official list of planned features that need help with development, please let me know where I can find it.

**Comment by crobibero** [comment]
We are at the tail end of migrating the database layer to use Entity Framework so I highly suggest holding off on starting any implementation until after that is complete (soon™️)

**Reply by Dan-613** [reply]
Are there any tasks that I could help out with on the Entity Framework migration?

**Reply by crobibero** [reply]
Work is happening [here](https://github.com/jellyfin/jellyfin/pull/12798), we are in the testing phase at the moment. 

Note that this change is not finalized so do not run it as your production instance if helping to test.

**Comment by thorbolo** [comment]
So the pre-Task ist merged.. @Dan-613 are you still considering to get your hands dirty on this? I would really love to this Feature :)
